gradlew

```diff
@@ -42,11 +42,6 @@ case "`uname`" in
     ;;
 esac
 
-# For Cygwin, ensure paths are in UNIX format before anything is touched.
```

Comment:
Out of curiosity, what prompted the changes to gradle wrapper?

Reply:
I think I opened a completely fresh Android Studio project and pasted the new `gradlew` over our starter project's `gradlew`

Reply:
But to answer your question specifically, not sure why things changed. I believe cywin is a windows platform thing.... FWIW